controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -154,28 +155,48 @@ public ConfigurationFile(final File configurationDir, final String rawName, fina
     /**
      * Creates a new ConfigurationFile.
      *
-     * @param configurationDir directory in which configuration files are stored. Cannot be {@code null} and must exist
-     *                         and be a directory
-     * @param rawName default name for configuration files of the type handled by this object.
-     *                Cannot be {@code null} or an empty string
-     * @param name user provided name of the configuration file to use
-     * @param interactionPolicy policy governing interaction with the configuration file
-     * @param useGit {@code true} if configuration is using Git to manage its history.
+     * @param configurationDir  directory in which configuration files are stored. Cannot be {@code null} and must exist
+     *                          and be a directory
+     * @param rawName           default name for configuration files of the type handled by this object.
+     *                          Cannot be {@code null} or an empty string
+     * @param name              user provided name of the configuration file to use
+     * @param interactionPolicy policy governing interaction with the configuration file.
+     * @param useGit            {@code true} if configuration is using Git to manage its history.
+     */
+    public ConfigurationFile(final File configurationDir, final String rawName, final String name, final InteractionPolicy interactionPolicy, final boolean useGit) {
+        this(configurationDir, rawName, name, interactionPolicy, useGit, null);
+    }
+
+    /**
+     * Creates a new ConfigurationFile.
+     *
+     * @param configurationDir  directory in which configuration files are stored. Cannot be {@code null} and must exist
+     *                          and be a directory.
+     * @param rawName           default name for configuration files of the type handled by this object.
+     *                          Cannot be {@code null} or an empty string.
+     * @param name              user provided name of the configuration file to use.
+     * @param interactionPolicy policy governing interaction with the configuration file.
+     * @param useGit            {@code true} if configuration is using Git to manage its history.
+     * @param tmpDir            The server temporal directory to use as a fallback if the configuration directory cannot
```

Comment:
Do you rather mean `temporary directory?`

Reply:
Yes @jmesnil , fixed the typo, thank you for your review